Auto update failing for btfs2

Hi, I'm Termux 🤖.

I'm here to help you update your Termux packages.

I've tried to update the btfs2 package, but it failed.

Here's the output of the update script:
<details>
<summary>Show log</summary>
<pre lang="bash">
INFO: Updating btfs2 [Current version: 2.3.2]
INFO: package being updated to 2.3.3.

  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100  656k    0  656k    0     0   931k      0 --:--:-- --:--:-- --:--:--  931k
100 6435k    0 6435k    0     0  3802k      0 --:--:--  0:00:01 --:--:-- 5844k
100 12.1M    0 12.1M    0     0  4718k      0 --:--:--  0:00:02 --:--:-- 6104k

You are about to commit these changes:

--------------------
diff --git a/packages/btfs2/build.sh b/packages/btfs2/build.sh
index bf238aac7..aaff4cea7 100644
--- a/packages/btfs2/build.sh
+++ b/packages/btfs2/build.sh
@@ -3,9 +3,9 @@ TERMUX_PKG_DESCRIPTION="Decentralized file system integrating with TRON network
 TERMUX_PKG_LICENSE="Apache-2.0, MIT"
 TERMUX_PKG_LICENSE_FILE="LICENSE-APACHE, LICENSE-MIT"
 TERMUX_PKG_MAINTAINER="Simbad Marino <cctechmx@gmail.com>"
-TERMUX_PKG_VERSION="2.3.2"
+TERMUX_PKG_VERSION="2.3.3"
 TERMUX_PKG_SRCURL=https://github.com/bittorrent/go-btfs/archive/refs/tags/btfs-v${TERMUX_PKG_VERSION}.tar.gz
-TERMUX_PKG_SHA256=4741c4d33ba9f241dbec4d3e71320f6ea0917486c4d6508959d0813d39e7594c
+TERMUX_PKG_SHA256=17f2367b2c6f50bf129bef595b15b504c25a0938d6ac27650ee07188d98da927
 TERMUX_PKG_AUTO_UPDATE=true
 TERMUX_PKG_UPDATE_VERSION_REGEXP="\d+\.\d+\.\d+"
 TERMUX_PKG_DEPENDS="libc++"
--------------------

bump(main/btfs2): 2.3.3


Not committing to Git!

INFO: Trying to build package.
INFO: Package btfs2 exists in main repo.
Running container 'termux-package-builder' from image 'ghcr.io/termux/package-builder'...
termux - building btfs2 for arch aarch64...
Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13134  100 13134    0     0  24902      0 --:--:-- --:--:-- --:--:-- 24875
100 13134  100 13134    0     0  24898      0 --:--:-- --:--:-- --:--:-- 24875
Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   1559      0 --:--:-- --:--:-- --:--:--  1559
gpg: Signature made Wed 27 Sep 2023 11:13:08 AM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13319  100 13319    0     0  25214      0 --:--:-- --:--:-- --:--:-- 25177
100 13319  100 13319    0     0  25210      0 --:--:-- --:--:-- --:--:-- 25177
Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   1565      0 --:--:-- --:--:-- --:--:--  1568
gpg: Signature made Thu 07 Sep 2023 12:20:57 PM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13120  100 13120    0     0  24446      0 --:--:-- --:--:-- --:--:-- 24432
100 13120  100 13120    0     0  24439      0 --:--:-- --:--:-- --:--:-- 24432
Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   1581      0 --:--:-- --:--:-- --:--:--  1583
gpg: Signature made Wed 27 Sep 2023 01:38:11 AM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading dependency libc++@25c if necessary...
Found libc++ in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libc++ to /home/builder/.termux-build/_cache-aarch64...
Downloading https://github.com/bittorrent/go-btfs/archive/refs/tags/btfs-v2.3.3.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100 40979    0 40979    0     0  72038      0 --:--:-- --:--:-- --:--:-- 72038
100 3477k    0 3477k    0     0  2259k      0 --:--:--  0:00:01 --:--:-- 3539k
100 10.7M    0 10.7M    0     0  4313k      0 --:--:--  0:00:02 --:--:-- 5534k
100 12.1M    0 12.1M    0     0  4129k      0 --:--:--  0:00:03 --:--:-- 5078k
Downloading https://golang.org/dl/go1.20.7.linux-amd64.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100    80  100    80    0     0    395      0 --:--:-- --:--:-- --:--:--   394

100    75  100    75    0     0    197      0 --:--:-- --:--:-- --:--:--   197

 34 95.5M   34 33.1M    0     0  30.1M      0  0:00:03  0:00:01  0:00:02 30.1M
 98 95.5M   98 94.1M    0     0  44.7M      0  0:00:02  0:00:02 --:--:-- 60.9M
100 95.5M  100 95.5M    0     0  45.2M      0  0:00:02  0:00:02 --:--:-- 61.7M
go: /home/builder/.termux-build/btfs2/src/go.mod already exists
go: downloading github.com/bittorrent/go-btfs-files v0.3.1
go: downloading github.com/ethereum/go-ethereum v1.11.1
go: downloading github.com/bittorrent/interface-go-btfs-core v0.8.2
go: downloading github.com/ipfs/go-log v1.0.5
go: downloading github.com/go-bindata/go-bindata/v3 v3.1.3
go: downloading github.com/ipfs/go-cid v0.4.0
go: downloading github.com/libp2p/go-libp2p v0.24.2
go: downloading github.com/bittorrent/go-btfs-api v0.5.0
go: downloading github.com/ip2location/ip2location-go/v9 v9.0.0
go: downloading github.com/multiformats/go-multiaddr v0.8.0
go: downloading github.com/ipfs/go-ipfs-blockstore v1.2.0
go: downloading github.com/ipfs/go-ipfs-pinner v0.2.1
go: downloading github.com/ipfs/go-ipld-format v0.4.0
go: downloading github.com/bittorrent/go-btfs-cmds v0.3.0
go: downloading github.com/bittorrent/go-btfs-common v0.9.0
go: downloading github.com/bittorrent/go-btfs-config v0.13.0-pre2
go: downloading github.com/tyler-smith/go-bip32 v1.0.0
go: downloading github.com/tyler-smith/go-bip39 v1.1.0
go: downloading golang.org/x/sys v0.6.0
go: downloading github.com/fsnotify/fsnotify v1.6.0
go: downloading github.com/jbenet/goprocess v0.1.4
go: downloading github.com/mitchellh/go-homedir v1.1.0
go: downloading github.com/bittorrent/go-mfs v0.4.0
go: downloading github.com/ipfs/go-blockservice v0.5.0
go: downloading github.com/ipfs/go-fetcher v1.6.1
go: downloading github.com/ipfs/go-filestore v1.2.0
go: downloading github.com/ipfs/go-graphsync v0.14.0
go: downloading github.com/ipfs/go-ipfs-exchange-interface v0.2.0
go: downloading github.com/ipfs/go-ipfs-provider v0.8.0
go: downloading github.com/ipfs/go-metrics-interface v0.0.1
go: downloading github.com/coreos/go-systemd/v22 v22.5.0
go: downloading github.com/hashicorp/go-multierror v1.1.1
go: downloading github.com/ipfs/go-ipfs-util v0.0.2
go: downloading github.com/ipfs/go-metrics-prometheus v0.0.2
go: downloading github.com/libp2p/go-libp2p-loggables v0.1.0
go: downloading github.com/libp2p/go-socket-activation v0.1.0
go: downloading github.com/mholt/archiver/v3 v3.3.0
go: downloading github.com/multiformats/go-multiaddr-dns v0.3.1
go: downloading github.com/pkg/errors v0.9.1
go: downloading github.com/prometheus/client_golang v1.15.1
go: downloading gopkg.in/yaml.v2 v2.4.0
go: downloading gotest.tools v2.2.0+incompatible
go: downloading github.com/libp2p/go-libp2p-kad-dht v0.20.0
go: downloading github.com/libp2p/go-libp2p-pubsub v0.8.1
go: downloading github.com/libp2p/go-libp2p-pubsub-router v0.6.0
go: downloading github.com/libp2p/go-libp2p-record v0.2.0
go: downloading go.uber.org/fx v1.18.2
go: downloading github.com/ipfs/go-datastore v0.6.0
go: downloading github.com/google/martian v2.1.0+incompatible
go: downloading go4.org v0.0.0-20200411211856-f5505b9728dd
go: downloading golang.org/x/net v0.7.0
go: downloading github.com/ipfs/go-cidutil v0.1.0
go: downloading github.com/multiformats/go-multibase v0.1.1
go: downloading github.com/anacrolix/torrent v1.52.5
go: downloading github.com/cheggaaa/pb v1.0.29
go: downloading github.com/ipfs/go-block-format v0.1.2
go: downloading github.com/ipfs/go-merkledag v0.8.1
go: downloading github.com/bittorrent/go-path v0.4.1
go: downloading github.com/ipld/go-car v0.4.0
go: downloading github.com/ipld/go-car/v2 v2.4.0
go: downloading github.com/ipld/go-ipld-prime v0.19.0
go: downloading github.com/multiformats/go-multihash v0.2.1
go: downloading gopkg.in/cheggaaa/pb.v1 v1.0.28
go: downloading github.com/bittorrent/go-unixfs v0.7.0
go: downloading github.com/bradfitz/iter v0.0.0-20191230175014-e8f45d346db8
go: downloading github.com/cenkalti/backoff/v4 v4.1.3
go: downloading github.com/cenkalti/backoff v2.2.1+incompatible
go: downloading github.com/dustin/go-humanize v1.0.0
go: downloading github.com/elgris/jsondiff v0.0.0-20160530203242-765b5c24c302
go: downloading github.com/ipfs/go-bitswap v0.11.0
go: downloading github.com/ipfs/go-ipfs-exchange-offline v0.3.0
go: downloading github.com/ipfs/go-verifcid v0.0.2
go: downloading github.com/libp2p/go-libp2p-kbucket v0.5.0
go: downloading github.com/multiformats/go-multicodec v0.8.1
go: downloading github.com/whyrusleeping/go-sysinfo v0.0.0-20190219211824-4a357d4b90b1
go: downloading github.com/whyrusleeping/multiaddr-filter v0.0.0-20160516205228-e903e4adabd7
go: downloading github.com/whyrusleeping/tar-utils v0.0.0-20201201191210-20a61371de5b
go: downloading github.com/bittorrent/go-btns v0.2.0
go: downloading github.com/libp2p/go-testutil v0.1.0
go: downloading github.com/alecthomas/units v0.0.0-20211218093645-b94a6e3cc137
go: downloading github.com/bittorrent/go-common/v2 v2.4.0
go: downloading github.com/google/uuid v1.3.0
go: downloading github.com/bittorrent/go-btfs-chunker v0.4.0
go: downloading github.com/gogo/protobuf v1.3.2
go: downloading github.com/shirou/gopsutil/v3 v3.20.12
go: downloading github.com/shirou/gopsutil v3.21.4-0.20210419000835-c7a38de76ee5+incompatible
go: downloading github.com/klauspost/reedsolomon v1.9.14
go: downloading github.com/bittorrent/protobuf v1.4.0
go: downloading github.com/orcaman/concurrent-map v0.0.0-20190826125027-8c72a8bb44f6
go: downloading github.com/looplab/fsm v0.1.0
go: downloading github.com/stretchr/testify v1.8.2
go: downloading go.uber.org/zap v1.24.0
go: downloading golang.org/x/sync v0.1.0
go: downloading github.com/ipfs/go-ipld-cbor v0.0.5
go: downloading github.com/bittorrent/go-eccrypto v0.1.0
go: downloading github.com/ipfs/go-ipfs-routing v0.3.0
go: downloading contrib.go.opencensus.io/exporter/prometheus v0.4.2
go: downloading github.com/libp2p/go-libp2p-http v0.4.0
go: downloading go.opencensus.io v0.24.0
go: downloading go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp v0.41.1
go: downloading github.com/cespare/xxhash v1.1.0
go: downloading github.com/gabriel-vasile/mimetype v1.4.1
go: downloading github.com/ipfs/go-ipfs-redirects-file v0.1.1
go: downloading github.com/ipfs/go-unixfsnode v1.4.0
go: downloading github.com/ipld/go-codec-dagpb v1.4.1
go: downloading github.com/libp2p/go-doh-resolver v0.4.0
go: downloading github.com/libp2p/go-libp2p-routing-helpers v0.4.0
go: downloading github.com/miekg/dns v1.1.50
go: downloading go.opentelemetry.io/otel v1.15.1
go: downloading go.opentelemetry.io/otel/trace v1.15.1
go: downloading go.uber.org/multierr v1.9.0
go: downloading github.com/ipfs/go-ipfs-posinfo v0.0.1
go: downloading github.com/libp2p/go-libp2p-testing v0.12.0
go: downloading github.com/benbjohnson/clock v1.3.0
go: downloading github.com/ipfs/go-log/v2 v2.5.1
go: downloading github.com/pbnjay/memory v0.0.0-20210728143218-7b4eea64cf58
go: downloading bazil.org/fuse v0.0.0-20200117225306-7b5117fecadc
go: downloading golang.org/x/crypto v0.6.0
go: downloading github.com/blang/semver v3.5.1+incompatible
go: downloading github.com/ipfs/go-detect-race v0.0.1
go: downloading github.com/markbates/pkger v0.17.0
go: downloading gopkg.in/natefinch/lumberjack.v2 v2.0.0
go: downloading github.com/bren2010/proquint v0.0.0-20160323162903-38337c27106d
go: downloading github.com/hashicorp/golang-lru v0.5.5-0.20210104140557-80c98217689d
go: downloading github.com/jbenet/go-is-domain v1.0.5
go: downloading github.com/whyrusleeping/base32 v0.0.0-20170828182744-c30ac30633cc
go: downloading github.com/ipfs/go-ipfs-ds-help v1.1.0
go: downloading github.com/jbenet/go-temp-err-catcher v0.1.0
go: downloading github.com/opentracing/opentracing-go v1.2.0
go: downloading github.com/ipfs/go-ds-badger v0.3.0
go: downloading github.com/ipfs/go-ds-flatfs v0.5.1
go: downloading github.com/ipfs/go-ipld-git v0.1.1
go: downloading github.com/ipfs/go-ds-leveldb v0.5.0
go: downloading github.com/syndtr/goleveldb v1.0.1-0.20210819022825-2ae1ddf74ef7
go: downloading github.com/golang/protobuf v1.5.3
go: downloading github.com/ipfs/go-ds-measure v0.2.0
go: downloading github.com/ipfs/go-fs-lock v0.0.7
go: downloading github.com/ipfs/go-delegated-routing v0.7.0
go: downloading github.com/rs/cors v1.7.0
go: downloading github.com/aws/aws-sdk-go v1.45.2
go: downloading github.com/gorilla/mux v1.7.3
go: downloading github.com/json-iterator/go v1.1.12
go: downloading github.com/vmihailenco/msgpack/v4 v4.3.12
go: downloading github.com/patrickmn/go-cache v2.1.0+incompatible
go: downloading github.com/status-im/keycard-go v0.2.0
go: downloading github.com/jbenet/go-random v0.0.0-20190219211222-123a90aedc0c
go: downloading github.com/ethersphere/go-sw3-abi v0.4.0
go: downloading github.com/btcsuite/btcd v0.22.1
go: downloading github.com/crackcomm/go-gitignore v0.0.0-20170627025303-887ab5e44cc3
go: downloading github.com/kisielk/errcheck v1.5.0
go: downloading golang.org/x/lint v0.0.0-20210508222113-6edffad5e616
go: downloading github.com/multiformats/go-varint v0.0.7
go: downloading github.com/mr-tron/base58 v1.2.0
go: downloading github.com/ipfs/bbloom v0.0.4
go: downloading go.uber.org/atomic v1.10.0
go: downloading github.com/btcsuite/btcutil v1.0.3-0.20201208143702-a53e38424cce
go: downloading github.com/decred/dcrd/dcrec/secp256k1/v4 v4.1.0
go: downloading google.golang.org/grpc v1.53.0
go: downloading github.com/FactomProject/basen v0.0.0-20150613233007-fe3947df716e
go: downloading github.com/FactomProject/btcutilecc v0.0.0-20130527213604-d3a63a5752ec
go: downloading github.com/libp2p/go-openssl v0.1.0
go: downloading github.com/minio/sha256-simd v1.0.0
go: downloading github.com/multiformats/go-multiaddr-fmt v0.1.0
go: downloading github.com/cheekybits/is v0.0.0-20150225183255-68e9c0620927
go: downloading go.uber.org/goleak v1.1.12
go: downloading github.com/flynn/noise v1.0.0
go: downloading github.com/libp2p/go-buffer-pool v0.1.0
go: downloading github.com/ipfs/go-ipfs-blocksutil v0.0.1
go: downloading github.com/Kubuxu/go-os-helper v0.0.1
go: downloading github.com/texttheater/golang-levenshtein v0.0.0-20180516184445-d188e65d659e
go: downloading github.com/facebookgo/atomicfile v0.0.0-20151019160806-2de1f203e7d5
go: downloading github.com/hashicorp/errwrap v1.1.0
go: downloading github.com/libp2p/go-libp2p-core v0.20.1
go: downloading github.com/andybalholm/brotli v1.0.4
go: downloading github.com/dsnet/compress v0.0.1
go: downloading github.com/golang/snappy v0.0.4
go: downloading github.com/klauspost/compress v1.15.15
go: downloading github.com/klauspost/pgzip v1.2.1
go: downloading github.com/nwaples/rardecode v1.0.0
go: downloading github.com/pierrec/lz4 v2.6.1+incompatible
go: downloading github.com/ulikunitz/xz v0.5.6
go: downloading github.com/xi2/xz v0.0.0-20171230120015-48954b6210f8
go: downloading github.com/beorn7/perks v1.0.1
go: downloading github.com/cespare/xxhash/v2 v2.2.0
go: downloading github.com/prometheus/client_model v0.3.0
go: downloading github.com/prometheus/common v0.42.0
go: downloading github.com/prometheus/procfs v0.9.0
go: downloading google.golang.org/protobuf v1.30.0
go: downloading gopkg.in/check.v1 v1.0.0-20201130134442-10cb98267c6c
go: downloading github.com/jbenet/go-cienv v0.1.0
go: downloading github.com/google/go-cmp v0.5.9
go: downloading github.com/libp2p/go-msgio v0.2.0
go: downloading github.com/whyrusleeping/timecache v0.0.0-20160911033111-cfcb2f1abfee
go: downloading github.com/libp2p/go-flow-metrics v0.1.0
go: downloading github.com/libp2p/zeroconf/v2 v2.2.0
go: downloading github.com/libp2p/go-netroute v0.2.1
go: downloading github.com/multiformats/go-multistream v0.3.3
go: downloading go.uber.org/dig v1.15.0
go: downloading github.com/multiformats/go-base32 v0.1.0
go: downloading github.com/multiformats/go-base36 v0.2.0
go: downloading github.com/anacrolix/missinggo v1.3.0
go: downloading github.com/anacrolix/dht/v2 v2.19.2-0.20221121215055-066ad8494444
go: downloading github.com/frankban/quicktest v1.14.4
go: downloading github.com/RoaringBitmap/roaring v1.2.3
go: downloading github.com/anacrolix/chansync v0.3.0
go: downloading github.com/anacrolix/generics v0.0.0-20230428105757-683593396d68
go: downloading github.com/anacrolix/go-libutp v1.3.1
go: downloading github.com/anacrolix/log v0.14.0
go: downloading github.com/anacrolix/missinggo/v2 v2.7.2-0.20230527121029-a582b4f397b9
go: downloading github.com/anacrolix/missinggo/perf v1.0.0
go: downloading github.com/anacrolix/multiless v0.3.0
go: downloading github.com/anacrolix/sync v0.4.0
go: downloading github.com/anacrolix/upnp v0.1.3-0.20220123035249-922794e51c96
go: downloading github.com/anacrolix/utp v0.1.0
go: downloading github.com/bahlo/generic-list-go v0.2.0
go: downloading github.com/davecgh/go-spew v1.1.1
go: downloading github.com/google/btree v1.1.2
go: downloading github.com/gorilla/websocket v1.5.0
go: downloading github.com/lispad/go-generics-tools v1.1.0
go: downloading github.com/pion/datachannel v1.5.2
go: downloading github.com/pion/webrtc/v3 v3.1.42
go: downloading golang.org/x/exp v0.0.0-20230206171751-46f607a40771
go: downloading golang.org/x/time v0.0.0-20220922220347-f3bd1da661af
go: downloading github.com/anacrolix/envpprof v1.2.1
go: downloading crawshaw.io/sqlite v0.3.3-0.20220618202545-d1964889ea3c
go: downloading github.com/edsrzf/mmap-go v1.1.0
go: downloading go.etcd.io/bbolt v1.3.6
go: downloading github.com/mattn/go-runewidth v0.0.9
go: downloading github.com/fatih/color v1.9.0
go: downloading github.com/mattn/go-colorable v0.1.13
go: downloading github.com/ipfs/go-ipld-legacy v0.1.1
go: downloading github.com/ipld/go-ipld-prime/storage/bsadapter v0.0.0-20211210234204-ce2a1c70cd73
go: downloading github.com/mgutz/ansi v0.0.0-20170206155736-9520e82c474b
go: downloading github.com/ipfs/go-ipfs-delay v0.0.1
go: downloading github.com/google/gopacket v1.1.19
go: downloading github.com/grpc-ecosystem/go-grpc-prometheus v1.2.0
go: downloading github.com/hypnoglow/go-pg-monitor v0.1.0
go: downloading github.com/hypnoglow/go-pg-monitor/gopgv9 v0.1.0
go: downloading github.com/whyrusleeping/chunker v0.0.0-20181014151217-fe64bd25879f
go: downloading github.com/klauspost/cpuid/v2 v2.2.4
go: downloading github.com/pmezard/go-difflib v1.0.0
go: downloading gopkg.in/yaml.v3 v3.0.1
go: downloading github.com/polydawn/refmt v0.0.0-20201211092308-30ac6d18308e
go: downloading github.com/whyrusleeping/cbor-gen v0.0.0-20210219115102-f37d292932f2
go: downloading github.com/fomichev/secp256k1 v0.0.0-20180413221153-00116ff8c62f
go: downloading github.com/warpfork/go-wish v0.0.0-20200122115046-b9ea61034e4a
go: downloading github.com/lucas-clemente/quic-go v0.31.1
go: downloading github.com/golang/mock v1.6.0
go: downloading github.com/prometheus/statsd_exporter v0.22.7
go: downloading github.com/libp2p/go-libp2p-gostream v0.5.0
go: downloading github.com/OneOfOne/xxhash v1.2.2
go: downloading github.com/spaolacci/murmur3 v1.1.0
go: downloading github.com/felixge/httpsnoop v1.0.3
go: downloading go.opentelemetry.io/otel/metric v0.38.1
go: downloading github.com/ucarion/urlpath v0.0.0-20200424170820-7ccc79b76bbb
go: downloading github.com/tj/assert v0.0.3
go: downloading golang.org/x/tools v0.3.0
go: downloading github.com/go-logr/logr v1.2.4
go: downloading github.com/go-logr/stdr v1.2.2
go: downloading github.com/petar/GoLLRB v0.0.0-20210522233825-ae3b015fd3e9
go: downloading github.com/whyrusleeping/cbor v0.0.0-20171005072247-63513f603b11
go: downloading github.com/ipfs/go-peertaskqueue v0.8.0
go: downloading github.com/ipfs/go-ipfs-chunker v0.0.5
go: downloading github.com/ipfs/go-ipfs-files v0.0.8
go: downloading github.com/ipfs/go-unixfs v0.3.1
go: downloading github.com/mattn/go-isatty v0.0.17
go: downloading github.com/libp2p/go-libp2p-xor v0.1.0
go: downloading github.com/libp2p/go-mplex v0.7.0
go: downloading github.com/libp2p/go-yamux/v4 v4.0.0
go: downloading github.com/raulk/go-watchdog v1.3.0
go: downloading github.com/libp2p/go-libp2p-asn-util v0.2.0
go: downloading github.com/libp2p/go-reuseport v0.2.0
go: downloading github.com/marten-seemann/tcp v0.0.0-20210406111302-dfbc87cc63fd
go: downloading github.com/mikioh/tcpinfo v0.0.0-20190314235526-30a79bb1804b
go: downloading github.com/marten-seemann/webtransport-go v0.4.3
go: downloading github.com/BurntSushi/toml v1.2.0
go: downloading github.com/dgraph-io/badger v1.6.2
go: downloading github.com/alexbrainman/goissue34681 v0.0.0-20191006012335-3fc7a47baff5
go: downloading github.com/ipfs/go-ipns v0.3.0
go: downloading github.com/ipld/edelweiss v0.2.0
go: downloading github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd
go: downloading github.com/modern-go/reflect2 v1.0.2
go: downloading github.com/holiman/big v0.0.0-20221017200358-a027dc42d04e
go: downloading github.com/vmihailenco/tagparser v0.1.2
go: downloading google.golang.org/appengine v1.6.7
go: downloading github.com/StackExchange/wmi v0.0.0-20190523213315-cbe66965904d
go: downloading github.com/onsi/ginkgo v1.16.5
go: downloading github.com/onsi/gomega v1.24.0
go: downloading github.com/btcsuite/btcd/chaincfg/chainhash v1.0.1
go: downloading github.com/btcsuite/btcd/btcec/v2 v2.3.2
go: downloading google.golang.org/genproto v0.0.0-20230221151758-ace64dc21148
go: downloading github.com/deckarep/golang-set/v2 v2.1.0
go: downloading gopkg.in/natefinch/npipe.v2 v2.0.0-20160621034901-c1b8fa8bdcce
go: downloading github.com/holiman/uint256 v1.2.0
go: downloading github.com/golang-jwt/jwt/v4 v4.3.0
go: downloading github.com/prometheus/tsdb v0.7.1
go: downloading github.com/cmars/basen v0.0.0-20150613233007-fe3947df716e
go: downloading launchpad.net/gocheck v0.0.0-20140225173054-000000000087
go: downloading github.com/decred/dcrd/crypto/blake256 v1.0.0
go: downloading github.com/mattn/go-pointer v0.0.1
go: downloading github.com/spacemonkeygo/spacelog v0.0.0-20180420211403-2296661a0572
go: downloading golang.org/x/term v0.5.0
go: downloading github.com/matttproud/golang_protobuf_extensions v1.0.4
go: downloading github.com/kr/pretty v0.3.1
go: downloading github.com/libp2p/go-cidranger v1.1.0
go: downloading github.com/libp2p/go-nat v0.1.0
go: downloading github.com/huandu/xstrings v1.3.2
go: downloading github.com/bits-and-blooms/bitset v1.2.2
go: downloading github.com/mschoch/smat v0.2.0
go: downloading github.com/rs/dnscache v0.0.0-20211102005908-e0241e321417
go: downloading github.com/anacrolix/mmsg v1.0.0
go: downloading github.com/anacrolix/lsan v0.0.0-20211126052245-807000409a62
go: downloading github.com/ajwerner/btree v0.0.0-20211221152037-f427b3e689c0
go: downloading github.com/tidwall/btree v1.6.0
go: downloading github.com/pion/logging v0.2.2
go: downloading github.com/pion/sctp v1.8.2
go: downloading github.com/pion/transport v0.13.1
go: downloading github.com/pion/dtls/v2 v2.2.4
go: downloading github.com/pion/ice/v2 v2.2.6
go: downloading github.com/pion/interceptor v0.1.11
go: downloading github.com/pion/randutil v0.1.0
go: downloading github.com/pion/rtcp v1.2.9
go: downloading github.com/pion/rtp v1.7.13
go: downloading github.com/pion/sdp/v3 v3.0.5
go: downloading github.com/pion/srtp/v2 v2.0.9
go: downloading crawshaw.io/iox v0.0.0-20181124134642-c51c3df30797
go: downloading github.com/warpfork/go-testmark v0.10.0
go: downloading github.com/ipfs/go-bitfield v1.1.0
go: downloading github.com/whyrusleeping/go-keyspace v0.0.0-20160322163242-5b898ac5add1
go: downloading github.com/go-pg/migrations/v7 v7.1.11
go: downloading github.com/go-pg/pg/v9 v9.2.1
go: downloading github.com/go-redis/redis/v7 v7.4.1
go: downloading github.com/grpc-ecosystem/go-grpc-middleware v1.2.2
go: downloading github.com/smartystreets/goconvey v1.6.4
go: downloading golang.org/x/xerrors v0.0.0-20220609144429-65e65417b02f
go: downloading github.com/onsi/ginkgo/v2 v2.5.1
go: downloading github.com/go-kit/log v0.2.1
go: downloading github.com/golang/groupcache v0.0.0-20210331224755-41bb18bfe9da
go: downloading github.com/Stebalien/go-bitfield v0.0.1
go: downloading github.com/ipfs/go-ipfs-pq v0.0.2
go: downloading github.com/hannahhoward/go-pubsub v0.0.0-20200423002714-8d62886cc36e
go: downloading go.opentelemetry.io/otel/sdk v1.8.0
go: downloading github.com/elastic/gosigar v0.14.2
go: downloading github.com/containerd/cgroups v1.0.4
go: downloading github.com/opencontainers/runtime-spec v1.0.2
go: downloading github.com/mikioh/tcpopt v0.0.0-20190314235656-172688c1accc
go: downloading github.com/mikioh/tcp v0.0.0-20190314235350-803a9b46060c
go: downloading github.com/marten-seemann/qpack v0.3.0
go: downloading github.com/tv42/httpunix v0.0.0-20191220191345-2ba4b9c3382c
go: downloading github.com/gobuffalo/here v0.6.0
go: downloading github.com/dgraph-io/ristretto v0.0.2
go: downloading github.com/jmespath/go-jmespath v0.4.0
go: downloading github.com/olekukonko/tablewriter v0.0.5
go: downloading github.com/VictoriaMetrics/fastcache v1.6.0
go: downloading github.com/go-ole/go-ole v1.2.4
go: downloading github.com/go-stack/stack v1.8.1
go: downloading github.com/cespare/cp v0.1.0
go: downloading github.com/holiman/bloomfilter/v2 v2.0.3
go: downloading github.com/fjl/memsize v0.0.0-20190710130421-bcb5799ab5e5
go: downloading github.com/hashicorp/go-bexpr v0.1.10
go: downloading github.com/urfave/cli/v2 v2.17.2-0.20221006022127-8f469abc00aa
go: downloading github.com/huin/goupnp v1.0.3
go: downloading github.com/jackpal/go-nat-pmp v1.0.2
go: downloading github.com/davidlazar/go-crypto v0.0.0-20200604182044-b73af7476f6c
go: downloading github.com/kr/text v0.2.0
go: downloading github.com/rogpeppe/go-internal v1.9.0
go: downloading github.com/koron/go-ssdp v0.0.3
go: downloading github.com/urfave/cli v1.22.2
go: downloading filippo.io/edwards25519 v1.0.0-rc.1
go: downloading github.com/benbjohnson/immutable v0.3.0
go: downloading github.com/pion/transport/v2 v2.0.0
go: downloading github.com/pion/udp v0.1.4
go: downloading github.com/pion/mdns v0.0.5
go: downloading github.com/pion/stun v0.3.5
go: downloading github.com/pion/turn/v2 v2.0.8
go: downloading github.com/ryszard/goskiplist v0.0.0-20150312221310-2dfbae5fcf46
go: downloading lukechampine.com/blake3 v1.1.7
go: downloading github.com/cskr/pubsub v1.0.2
go: downloading mellium.im/sasl v0.3.1
go: downloading github.com/go-pg/zerochecker v0.2.0
go: downloading github.com/jinzhu/inflection v1.0.0
go: downloading github.com/segmentio/encoding v0.3.6
go: downloading github.com/francoispqt/gojay v1.2.13
go: downloading golang.org/x/mod v0.7.0
go: downloading github.com/jtolds/gls v4.20.0+incompatible
go: downloading github.com/smartystreets/assertions v1.0.0
go: downloading github.com/go-logfmt/logfmt v0.5.1
go: downloading github.com/docker/go-units v0.5.0
go: downloading github.com/godbus/dbus/v5 v5.1.0
go: downloading github.com/marten-seemann/qtls-go1-18 v0.1.3
go: downloading github.com/marten-seemann/qtls-go1-19 v0.1.1
go: downloading golang.org/x/text v0.7.0
go: downloading github.com/AndreasBriese/bbloom v0.0.0-20190825152654-46b345b51c96
go: downloading github.com/dgryski/go-farm v0.0.0-20190423205320-6a90982ecee2
go: downloading github.com/cockroachdb/pebble v0.0.0-20230209160836-829675f94811
go: downloading github.com/jmespath/go-jmespath/internal/testify v1.5.1
go: downloading github.com/nxadm/tail v1.4.8
go: downloading github.com/tklauser/go-sysconf v0.3.5
go: downloading github.com/gballet/go-libpcsclite v0.0.0-20190607065134-2772fd86a8ff
go: downloading github.com/mitchellh/pointerstructure v1.2.0
go: downloading github.com/cpuguy83/go-md2man/v2 v2.0.2
go: downloading github.com/xrash/smetrics v0.0.0-20201216005158-039620a65673
go: downloading github.com/cpuguy83/go-md2man v1.0.10
go: downloading github.com/anacrolix/stm v0.4.0
go: downloading github.com/vmihailenco/bufpool v0.1.11
go: downloading github.com/codemodus/kace v0.5.1
go: downloading github.com/gopherjs/gopherjs v0.0.0-20190910122728-9d188e94fb99
go: downloading github.com/go-task/slim-sprig v0.0.0-20210107165309-348f09dbbbc0
go: downloading github.com/segmentio/asm v1.2.0
go: downloading github.com/cockroachdb/errors v1.9.1
go: downloading github.com/cockroachdb/redact v1.1.3
go: downloading gopkg.in/tomb.v1 v1.0.0-20141024135613-dd632973f1e7
go: downloading github.com/tklauser/numcpus v0.2.2
go: downloading github.com/mitchellh/mapstructure v1.4.1
go: downloading github.com/russross/blackfriday/v2 v2.1.0
go: downloading github.com/google/pprof v0.0.0-20221203041831-ce31453925ec
go: downloading github.com/russross/blackfriday v1.5.2
go: downloading github.com/cockroachdb/logtags v0.0.0-20230118201751-21c54148d20b
go: downloading github.com/getsentry/sentry-go v0.18.0
go: downloading github.com/DataDog/zstd v1.5.2
go: downloading github.com/alecthomas/atomic v0.1.0-alpha2
go: downloading github.com/alecthomas/assert/v2 v2.0.0-alpha3
go: downloading github.com/hexops/gotextdiff v1.0.3
go: downloading github.com/alecthomas/repr v0.0.0-20210801044451-80ca428c5142
go version go1.20.7 linux/amd64
bin/check_go_version 1.18
plugin/loader/preload.sh > plugin/loader/preload.go
go fmt plugin/loader/preload.go >/dev/null
go build "-asmflags=all='-trimpath='" "-gcflags=all='-trimpath='" -ldflags="-X "github.com/bittorrent/go-btfs".CurrentCommit=" -o "cmd/btfs/btfs" "github.com/bittorrent/go-btfs/cmd/btfs"
# github.com/lucas-clemente/quic-go/internal/qtls
../../../go/pkg/mod/github.com/lucas-clemente/quic-go@v0.31.1/internal/qtls/go120.go:5:13: cannot use "The version of quic-go you're using can't be built on Go 1.20 yet. For more details, please see https://github.com/lucas-clemente/quic-go/wiki/quic-go-and-Go-versions." (untyped string constant "The version of quic-go you're using can't be built on Go 1.20 yet. F...) as int value in variable declaration
make: *** [cmd/btfs/Rules.mk:22: cmd/btfs/btfs] Error 1
ERROR: failed to build.</pre>
</details>

<hr>
<i>
Above error occured when I last tried to update at 2023-09-27 12:28:25 UTC.<br>
Run ID: <a href="https://github.com/termux/termux-packages/actions/runs/6325656313">6325656313</a><br><br>
<b>Note:</b> Automatic updates will be disabled until this issue is resolved.<br>
</i>

--- a/packages/btfs2/build.sh
+++ b/packages/btfs2/build.sh
@@ -3,19 +3,46 @@ TERMUX_PKG_DESCRIPTION="Decentralized file system integrating with TRON network 
 TERMUX_PKG_LICENSE="Apache-2.0, MIT"
 TERMUX_PKG_LICENSE_FILE="LICENSE-APACHE, LICENSE-MIT"
 TERMUX_PKG_MAINTAINER="Simbad Marino <cctechmx@gmail.com>"
-TERMUX_PKG_VERSION="2.3.2"
+TERMUX_PKG_VERSION="2.3.3"
 TERMUX_PKG_SRCURL=https://github.com/bittorrent/go-btfs/archive/refs/tags/btfs-v${TERMUX_PKG_VERSION}.tar.gz
-TERMUX_PKG_SHA256=4741c4d33ba9f241dbec4d3e71320f6ea0917486c4d6508959d0813d39e7594c
+TERMUX_PKG_SHA256=17f2367b2c6f50bf129bef595b15b504c25a0938d6ac27650ee07188d98da927
 TERMUX_PKG_AUTO_UPDATE=true
 TERMUX_PKG_UPDATE_VERSION_REGEXP="\d+\.\d+\.\d+"
 TERMUX_PKG_DEPENDS="libc++"
 TERMUX_PKG_BUILD_IN_SRC=true
 TERMUX_PKG_CONFLICTS="btfs"
 TERMUX_PKG_REPLACES="btfs"
-TERMUX_PKG_GO_USE_OLDER=true
+
+termux_setup_golang_119() {
+	if [ "$TERMUX_ON_DEVICE_BUILD" = "false" ]; then
+		local TERMUX_GO_VERSION=go1.19.13
+		local TERMUX_GO_SHA256=4643d4c29c55f53fa0349367d7f1bb5ca554ea6ef528c146825b0f8464e2e668
+		local TERMUX_GO_PLATFORM=linux-amd64
+
+		local TERMUX_BUILDGO_FOLDER="${TERMUX_PKG_CACHEDIR}/${TERMUX_GO_VERSION}"
+
+		export GOROOT=$TERMUX_BUILDGO_FOLDER
+		export PATH="${GOROOT}/bin:${PATH}"
+
+		if [ -d "$TERMUX_BUILDGO_FOLDER" ]; then return; fi
+
+		local TERMUX_BUILDGO_TAR=$TERMUX_PKG_CACHEDIR/${TERMUX_GO_VERSION}.${TERMUX_GO_PLATFORM}.tar.gz
+		rm -Rf "$TERMUX_PKG_CACHEDIR/go" "$TERMUX_BUILDGO_FOLDER"
+		termux_download https://golang.org/dl/${TERMUX_GO_VERSION}.${TERMUX_GO_PLATFORM}.tar.gz \
+			"$TERMUX_BUILDGO_TAR" \
+			"$TERMUX_GO_SHA256"
+
+		( cd "$TERMUX_PKG_CACHEDIR"; tar xf "$TERMUX_BUILDGO_TAR"; 
+		( cd go; . ${TERMUX_PKG_BUILDER_DIR}/fix-hardcoded-etc-resolv-conf.sh )
+		  mv go "$TERMUX_BUILDGO_FOLDER"; rm "$TERMUX_BUILDGO_TAR" )
+	else
+		termux_error_exit "This package cannot build on device currently."
+	fi
+}
 
 termux_step_pre_configure() {
-	termux_setup_golang
+	termux_setup_golang_119
+
 	go mod init || :
 	go mod tidy
 }

--- a/packages/btfs2/fix-hardcoded-etc-resolv-conf.diff
+++ b/packages/btfs2/fix-hardcoded-etc-resolv-conf.diff
@@ -1,0 +1,97 @@
+--- a/src/net/conf.go
++++ b/src/net/conf.go
+@@ -2,7 +2,7 @@
+ // Use of this source code is governed by a BSD-style
+ // license that can be found in the LICENSE file.
+
+-//go:build !js
++//go:build !js && !android
+
+ package net
+
+--- a/src/net/conf_android.go
++++ b/src/net/conf_android.go
+@@ -2,7 +2,7 @@
+ // Use of this source code is governed by a BSD-style
+ // license that can be found in the LICENSE file.
+
+-//go:build !js
++//go:build android
+
+ package net
+
+--- a/src/net/conf_android.go
++++ b/src/net/conf_android.go
+@@ -113,10 +113,10 @@ func initConfVal() {
+ 	}
+
+ 	if runtime.GOOS != "openbsd" {
+-		confVal.nss = parseNSSConfFile("/etc/nsswitch.conf")
++		confVal.nss = parseNSSConfFile("@TERMUX_PREFIX@/etc/nsswitch.conf")
+ 	}
+
+-	confVal.resolv = dnsReadConfig("/etc/resolv.conf")
++	confVal.resolv = dnsReadConfig("@TERMUX_PREFIX@/etc/resolv.conf")
+ 	if confVal.resolv.err != nil && !os.IsNotExist(confVal.resolv.err) &&
+ 		!os.IsPermission(confVal.resolv.err) {
+ 		// If we can't read the resolv.conf file, assume it
+@@ -126,7 +126,7 @@ func initConfVal() {
+ 		confVal.forceCgoLookupHost = true
+ 	}
+
+-	if _, err := os.Stat("/etc/mdns.allow"); err == nil {
++	if _, err := os.Stat("@TERMUX_PREFIX@/etc/mdns.allow"); err == nil {
+ 		confVal.hasMDNSAllow = true
+ 	}
+ }
+--- a/src/net/dnsclient_unix.go
++++ b/src/net/dnsclient_unix.go
+@@ -2,7 +2,7 @@
+ // Use of this source code is governed by a BSD-style
+ // license that can be found in the LICENSE file.
+
+-//go:build !js
++//go:build !js && !android
+
+ // DNS client: see RFC 1035.
+ // Has to be linked into package net for Dial.
+--- a/src/net/dnsclient_android.go
++++ b/src/net/dnsclient_android.go
+@@ -2,7 +2,7 @@
+ // Use of this source code is governed by a BSD-style
+ // license that can be found in the LICENSE file.
+
+-//go:build !js
++//go:build android
+
+ // DNS client: see RFC 1035.
+ // Has to be linked into package net for Dial.
+--- a/src/net/dnsclient_android.go
++++ b/src/net/dnsclient_android.go
+@@ -355,7 +355,7 @@ func (conf *resolverConfig) init() {
+ 	// resolv.conf twice the first time.
+ 	conf.dnsConfig = systemConf().resolv
+ 	if conf.dnsConfig == nil {
+-		conf.dnsConfig = dnsReadConfig("/etc/resolv.conf")
++		conf.dnsConfig = dnsReadConfig("@TERMUX_PREFIX@/etc/resolv.conf")
+ 	}
+ 	conf.lastChecked = time.Now()
+
+@@ -427,7 +427,7 @@ func (r *Resolver) lookup(ctx context.Context, name string, qtype dnsmessage.Typ
+ 		// For consistency with libc resolvers, report no such host.
+ 		return dnsmessage.Parser{}, "", &DNSError{Err: errNoSuchHost.Error(), Name: name, IsNotFound: true}
+ 	}
+-	resolvConf.tryUpdate("/etc/resolv.conf")
++	resolvConf.tryUpdate("@TERMUX_PREFIX@/etc/resolv.conf")
+ 	resolvConf.mu.RLock()
+ 	conf := resolvConf.dnsConfig
+ 	resolvConf.mu.RUnlock()
+@@ -604,7 +604,7 @@ func (r *Resolver) goLookupIPCNAMEOrder(ctx context.Context, network, name strin
+ 		// See comment in func lookup above about use of errNoSuchHost.
+ 		return nil, dnsmessage.Name{}, &DNSError{Err: errNoSuchHost.Error(), Name: name, IsNotFound: true}
+ 	}
+-	resolvConf.tryUpdate("/etc/resolv.conf")
++	resolvConf.tryUpdate("@TERMUX_PREFIX@/etc/resolv.conf")
+ 	resolvConf.mu.RLock()
+ 	conf := resolvConf.dnsConfig
+ 	resolvConf.mu.RUnlock()

--- a/packages/btfs2/fix-hardcoded-etc-resolv-conf.sh
+++ b/packages/btfs2/fix-hardcoded-etc-resolv-conf.sh
@@ -1,0 +1,12 @@
+for f in src/net/conf_android.go src/net/dnsclient_android.go; do
+	if [ -e "${f}" ]; then
+		termux_error_exit "Error: file ${f} already exists."
+	fi
+done
+
+cp -T src/net/conf.go src/net/conf_android.go
+cp -T src/net/dnsclient_unix.go src/net/dnsclient_android.go
+
+sed -e "s|@TERMUX_PREFIX@|$TERMUX_PREFIX|" \
+	${TERMUX_SCRIPTDIR}/packages/btfs2/fix-hardcoded-etc-resolv-conf.diff \
+	| patch --silent -p1

